data/json/professions.json

```diff
@@ -1062,8 +1063,11 @@
   {
     "type": "profession",
     "id": "football_player",
-    "name": "Football Player",
-    "description": "Your epic touchdown was ruined when your coach's guts were torn out by undead fans.  Now you're first pick in the apocalypse draft.",
+    "name": "Soccer Player",
+    "description": {
+      "male": "Soccer, rugby, American football — whatever you term it, you'd played it extensively before the Cataclysm and were quite the sportsman.  Now, your trade has left you with a set of football armour that you'd packed along during the evacuation, a strong throwing arm, and experience in pitch-side tussling.  With your ball at your side, the only keepsake you'd been able to hold on to through the end of days, you suppose that it's time to hustle.",
```

Comment:
Rugby/soccer/everywhere-but-US football doesn't wear this type of equipment or involve lots of throwing. Including them all under here doesn't make sense.